discovery/consul/consul.go

```diff
@@ -25,12 +25,14 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	consul "github.com/hashicorp/consul/api"
-	"github.com/mwitkow/go-conntrack"
+	conntrack "github.com/mwitkow/go-conntrack"
```

Comment:
Is this (and same in some other places) because your editor still has trouble with package names that are different from the last directory name component? It would be good to get to the bottom of that issue in general, as those aliases shouldn't be needed... then again, I'm not really opposed to them either, as it could make it easier for humans to see what is what.

Reply:
Found it: https://github.com/Microsoft/vscode-go/issues/613#issuecomment-353614168